src/seedu/addressbook/AddressBook.java

```diff
@@ -269,7 +282,7 @@ private static void processProgramArgs(String[] args) {
         if(args.length == 0) {
             setupDefaultFileForStorage();
         }
-    }
+    }*/
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.